core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -339,6 +339,7 @@ public enum BuiltInMethod {
   UPPER(SqlFunctions.class, "upper", String.class),
   LOWER(SqlFunctions.class, "lower", String.class),
   ASCII(SqlFunctions.class, "ascii", String.class),
+  CHAR(SqlFunctions.class, "charN", int.class),
   REPEAT(SqlFunctions.class, "repeat", String.class, int.class),
```

Comment:
should this be `CHAR(SqlFunctions.class, "charN", long.class)`?

and don't you also need `CHR(SqlFunctions.class, "chr", long.class)`?

Reply:
Thank you for your careful review, I changed to them.